bin/steps/pipenv

```diff
@@ -9,14 +9,16 @@ set -e
 if [[ -f Pipfile.lock ]]; then
     if [[ -f .heroku/python/Pipfile.lock.sha256 ]]; then
         if [[ $(openssl dgst -sha256 Pipfile.lock) == $(cat .heroku/python/Pipfile.lock.sha256) ]]; then
-            if [[ ! "$PIPENV_ALWAYS_INSTALL" ]]; then
+            # Measure that we're using Pipenv.
+            mcount "tool.pipenv"
+
+            # Don't skip installation of there are git deps.
+            if ! grep -q 'git' Pipfile.lock; then
```

Comment:
This would trigger even if a module name just contained `git` no?

Reply:
Also tests seem like they would be useful here

Reply:
this is a hotfix, needs to be released ASAP. Current testing framework doesn't allow for re-builds (which would trigger this). See above comments for justification on using 'git'

Reply:
> Current testing framework doesn't allow for re-builds (which would trigger this)

It's possible by passing in the cache directory manually, like these tests do:
https://github.com/heroku/heroku-buildpack-python/blob/7d975e74a93077f301916c85bce1089d59cb11f6/test/run#L85-L104

Reply:
happy to do that after releasing this — this is breaking customer builds at the moment.

Reply:
e.g. *emergency*